pkgs/tools/misc/nvimpager/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, pkgs
+# Defaults to false until I find out how to provide /dev/tty inside the
+# sandbox.
+, doCheck ? false
+}:
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "nvimpager";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "lucc";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1xy5387szfw0bp8dr7d4z33wd4xva7q219rvz8gc0vvv1vsy73va";
+  };
+
+  buildInputs = with pkgs; [
+    ncurses # for tput
+    neovim
+    procps # for nvim_get_proc() which uses ps(1)
+    pandoc
```

Comment:
AFAIU `pandoc` is used here to build your man-page. If that's the case, it should be in `nativeBuildInputs`. An explanation what this does (in contrast to `buildInputs`) can be found in the manual at https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-dependencies.